docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,529 @@
+sudo apt update
+sudo apt upgrade
+```
+
+Make sure you have `pip` installed for Python 3:
+
+``` shell
+pip --version
+```
+
+If not, run the following command:
+
+``` shell
+sudo apt install python3-pip
+```
+
+Run the following command while connected to your Pi with SSH to install [`Adafruit_CircuitPython_MCP3xxx`](https://github.com/adafruit/Adafruit_CircuitPython_MCP3xxx/):
+
+``` shell
+sudo pip3 install adafruit-circuitpython-mcp3xxx
+```
+
+Create a new directory for your plant watering robot's files and navigate to this directory in your terminal session.
+For example, run the following commands:
+
+``` shell
+mkdir plant-watering-robot
+cd plant-watering-robot
+```
+
+After navigating to this directory, create a new Python file called <file>adctesting.py</file> and open up the file.
+For example, run the following commands:
+
+``` shell
+touch adctesting.py
+nano adctesting.py
+```
+
+Now, add the following Python code to <file>adctesting.py</file> to test reading values from your capacitive soil moisture sensor through your MCP3008 ADC:
+
+``` python
+import time
+import busio
+import digitalio
+import board
+import adafruit_mcp3xxx.mcp3008 as MCP
+from adafruit_mcp3xxx.analog_in import AnalogIn
+
+# Create the SPI bus
+spi = busio.SPI(clock=board.SCK, MISO=board.MISO, MOSI=board.MOSI)
+
+# Create the cs (chip select)
+cs = digitalio.DigitalInOut(board.D25)
+
+# Create the MCP3008 object
+mcp = MCP.MCP3008(spi, cs)
+
+# Create an analog input channel on Pin 0
+chan = AnalogIn(mcp, MCP.P0)
+
+print('Reading MCP3008 values, press Ctrl-C to quit...')
+while True:
+    print('Raw ADC Value: ', chan.value)
+    time.sleep(1)
+```
+
+Run the code as follows:
+
+``` shell
+sudo python3 adctesting.py
+```
+
+{{% alert title="Tip" color="tip" %}}
+
+If running this code returns `ImportError: No module named busio`, try again after reinstalling `adafruit-blinka`:
+
+``` shell
+sudo pip3 install --force-reinstall adafruit-blinka
+```
+
+{{% /alert %}}
+
+Now, you should see the Moisture Sensor values outputted by the MCP3008 in the range of `0` to `1023`.
+Test your sensor by putting it in air, water, and different soils to see how the values change.
+
+![Terminal output of capacitive soil moisture sensor values.](../../img/plant-watering-pi/moisture-sensor-output.png)
+
+### Configure the Components of your Robot in the Viam app
+
+Follow [this guide](/installation/install/) to install `viam-server` on your pi, create a new robot, and connect to it on [the Viam app](https://app.viam.com).
+
+Then, navigate to your new robot's page on the app and click on the **CONFIG** tab.
+
+First, add your Pi as a [board component](/components/board/) by creating a new component with **type** `board` and **model** `pi`:
+
+{{< tabs name="Configure an Pi Board" >}}
+{{% tab name="Config Builder" %}}
+
+<img src="../../img/plant-watering-pi/pi-board-config-builder.png" alt="Creation of a pi board in the Viam app config builder." style="max-width:800px" />
```

Comment:
I'm not entirely sure why specifying max-width on these images makes it ignore the available width in the tab but I think this shoudl fix it. If not, just not specifiying max-width seemed to do the right thing for me
```suggestion
![Creation of a pi board in the Viam app config builder.](../../img/plant-watering-pi/pi-board-config-builder.png)
```